daemon/info.go

```diff
@@ -181,7 +181,7 @@ func (daemon *Daemon) SystemVersion() types.Version {
 		},
 
 		// Populate deprecated fields for older clients
-		Engine:        "balena",
+		Engine:        "balenadctl",
```

Comment:
This identifier should be of the daemon's (or even the project's as a whole), it seems you're assigning the client's. I think `balena-engine` or `balenaEngine` is more appropriate here.

Reply:
@dfunckt Akis, indeed, well spotted! I have changed it to `balena-engine` as suggested, though after the latest renaming, it's actually still the client name:

From | To
--- | ---
balena | balena-engine
balenad | balena-engine-daemon
balena-containerd | balena-engine-containerd
balena-containerd-shim | balena-engine-containerd-shim
balena-containerd-ctr | balena-engine-containerd-ctr
balena-runc | balena-engine-runc
balena-proxy | balena-engine-proxy
balena-init | balena-engine-init

But unlike `balenadctl`, `balena-engine` is now also the prefix of all executables and also the service name of the OS startup scripts (like upstart, init.d...), so more reasonable as the daemon engine name here. The alternatives would be `balenaEngine` or `balena-engine-daemon`.